### PR TITLE
basic styling for new movie, added margins, text decoration for heading,...

### PR DESCRIPTION
**Pull request by DuncanRose**
... size of viewing screen

### CODE REVIEW

File: app/controllers/movies_controller.rb [hunk 1]
```diff
@@ -3,6 +3,15 @@ class MoviesController < ApplicationController
...
     @movies = Movie.all.limit(10)
+
+    @data = File.read('app/assets/javascripts/feature_presentation.json')
+
+    # binding.pry
+
+    # respond_to do |format|
+    #   format.js { render :json }
+    # end
+
```
Comment (cmkoller) [inline app/controllers/movies_controller.rb]: delete commented out content

File: app/controllers/movies_controller.rb [hunk 2]
```diff
@@ -11,6 +20,7 @@ def show
 
   def new
     @movie = Movie.new
+    @feature = "Add"
```
Comment (cmkoller) [inline app/controllers/movies_controller.rb]: it doesn't look like you're using this instance variable in your view - so what's it doing here?

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -9,22 +9,15 @@
...
  *= require_tree .
-
+ *= require_self
 
  */
 
 body {
-  background-image: url(http://scanachmedia.com/wp-content/uploads/2013/12/COLD-EDGE-BACKGROUND.jpg);
-  background-size: 100% 100%;
+  background-color: #3C0201;
```
Comment (cmkoller) [inline app/assets/stylesheets/application.scss]: Now that this code has gotten a lot longer, it should really go into a separate file in this folder called something like `style.scss`

File: app/controllers/movies_controller.rb (hunk 2 above)
Comment (DuncanRose) [inline app/controllers/movies_controller.rb]: This has been deleted...

File: app/assets/stylesheets/application.scss (hunk 3 above)
Comment (DuncanRose) [inline app/assets/stylesheets/application.scss]: Done. There is now a Movies.css file

**Pull request merged**